src/main/resources/i18n/Enums_es.properties

```diff
@@ -0,0 +1,141 @@
+public.FacilityConnectionState.NotConnected=No conectado
+public.FacilityType.Desalination=Desalinización
+public.FacilityType.Nursery=Vivero
+public.FacilityType.ReverseOsmosis=Ósmosis inversa
+public.FacilityType.SeedBank=Banco de semillas
+public.GrowthForm.Fern=Helecho
+public.GrowthForm.Forb=Forb
+public.GrowthForm.Graminoid=Graminoide
+public.GrowthForm.Shrub=Arbusto
+public.GrowthForm.Tree=Árbol
+public.NotificationCriticality.Error=Error
+public.NotificationCriticality.Info=Información
+public.NotificationCriticality.Success=Éxito
+public.NotificationCriticality.Warning=Advertencia
+public.NotificationType.AccessionScheduledToEndDrying=Adhesión programada para finalizar el secado
+public.NotificationType.DeviceUnresponsive=El dispositivo no responde
+public.NotificationType.FacilityAlertRequested=Alerta de instalación solicitada
+public.NotificationType.FacilityIdle=Instalación inactiva
+public.NotificationType.NurserySeedlingBatchReady=Lote de plántulas de vivero listo
+public.NotificationType.ReportCreated=Informe creado
+public.NotificationType.SensorOutOfBounds=Sensor fuera de límites
+public.NotificationType.UnknownAutomationTriggered=Automatización desconocida activada
+public.NotificationType.UserAddedToOrganization=Usuario agregado a la organización
+public.ReportStatus.InProgress=En curso
+public.ReportStatus.Locked=Bloqueado
+public.ReportStatus.New=Nuevo
+public.ReportStatus.Submitted=Enviado
+public.Role.Admin=Administrador
+public.Role.Contributor=Colaborador
+public.Role.Manager=Gerente
+public.Role.Owner=Propietario
+public.SeedStorageBehavior.Intermediate=Intermediario
+public.SeedStorageBehavior.Orthodox=Ortodoxo
+public.SeedStorageBehavior.Recalcitrant=Recalcitrante
+public.SeedStorageBehavior.Unknown=Desconocido
+public.SpeciesProblemField.ScientificName=Nombre científico
+public.SpeciesProblemType.NameIsSynonym=El nombre es un sinónimo
+public.SpeciesProblemType.NameMisspelled=Nombre mal escrito
+public.SpeciesProblemType.NameNotFound=Nombre no encontrado
+public.TimeseriesType.Numeric=Numérico
+public.TimeseriesType.Text=Texto
+public.UploadProblemType.DuplicateValue=Valor duplicado
+public.UploadProblemType.MalformedValue=Valor mal formado
+public.UploadProblemType.MissingRequiredValue=Falta el valor requerido
+public.UploadProblemType.UnrecognizedValue=Valor no reconocido
+public.UploadStatus.AwaitingProcessing=Pendiente de tramitación
+public.UploadStatus.AwaitingUserAction=Pendiente de la acción del usuario
+public.UploadStatus.AwaitingValidation=Pendiente de validación
+public.UploadStatus.Completed=Completado
+public.UploadStatus.Invalid=No válido
+public.UploadStatus.Processing=Procesando
+public.UploadStatus.ProcessingFailed=Procesamiento fallido
+public.UploadStatus.Receiving=Recibiendo
+public.UploadStatus.ReceivingFailed=Recepción fallida
+public.UploadStatus.Validating=Validando
+public.UploadType.AccessionCSV=Acceso CSV
+public.UploadType.SeedlingBatchCSV=Lote de siembra CSV
```

Comment:
If we are translating "seedling" as "plántula" is should be "Lote de plántulas" here